common/src/main/java/com/wynntils/features/inventory/MythicBoxScalerFeature.java

```diff
@@ -14,7 +14,7 @@
 import com.wynntils.utils.wynn.WynnItemMatchers;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-@ConfigCategory(Category.COMBAT)
+@ConfigCategory(Category.INVENTORY)
 public class MythicBoxScalerFeature extends Feature {
```

Comment:
Same here, but worse. This deals with items on the ground. That is not "Inventory" in any way.